libs/ui-lib/lib/ocm/components/clusterConfiguration/OcmClusterDetailsFormFields.tsx

```diff
@@ -113,10 +113,15 @@ export const OcmClusterDetailsFormFields = ({
   }, []);
 
   React.useEffect(() => {
-    if (!oracleDropdownItemState?.isSupported) {
-      setFieldValue('platform', 'none');
+    if (clusterPlatform !== undefined) {
+      const platform = clusterPlatform === 'baremetal' ? 'none' : clusterPlatform;
+      setFieldValue('platform', platform);
+    } else {
+      if (!oracleDropdownItemState?.isSupported) {
+        setFieldValue('platform', 'none');
```

Comment:
Given the new discussion, would it not be `baremetal` the best value to set here?